docs/installation.rst

```diff
@@ -327,6 +327,9 @@ Virtualenv_ can be used to build and install Borg without affecting
 the system Python or requiring root access.  Using a virtual environment is
 optional, but recommended except for the most simple use cases.
 
+Ensure to install the dependencies as described within :ref:`source-install`.
+(Quick-links: :ref:`debian-ubuntu`, :ref:`fedora`, :ref:`macos`, :ref:`freebsd`, :ref:`cygwin`)
```

Comment:
Please remove the second line, we do not want to maintain all these links.

Making the docs longer and more complex is not helpful. As you have noticed, people don't like to read long docs.

Reply:
> we do not want to maintain all these links.

those links are static, in-doc, zero maintenance.

> Making the docs longer and more complex is not helpful. 

The docs are too long  (for target audience: developer) , they need reduction.

> As you have noticed, people don't like to read long docs.

That's why I don't point them to line 143/From Source, forcing them to start reading redundant stuff about "Some older Linux systems...". You can do that (and pay the bounty proudly to yourself). I wont do that.

The solution is:
* point to ubuntu (luckily and logically at the start, as widelly used)
* point to 143/From Source, but provide quicklinks
* alternatively add the quick-links section right at the start of 143/From source

I can assist you to fix docs issues (it's a typical "python-domain" mess), mainly by simplifying things.

(Alternatively you can wait for a "code-monkey" to pickup the bounties. They usually do not care about quality, and love to get directions.)

Reply:
no, they are not zero maintenance:

if we add or remove OSes, we would have to update that list. also, in case some more recent release of debian/ubuntu behaves different from an older one: we would have to update that list.

pointing them to the right section of the docs is a good idea, but then they can find their OS themselves there.

Reply:
> if we add or remove OSes, we would have to update that list. also, in case some more recent release of debian/ubuntu behaves different from an older one: we would have to update that list.

This is called "usual work load"

> but then they can find their OS themselves there.

Oh, they can find everything themselves (yelling "who tf has written this manuals"? while at it).